tests/validation/tests/v3_api/test_hosted_eks_cluster.py

```diff
@@ -190,6 +193,35 @@ def test_eks_v2_create_import_cluster():
     create_and_validate_eks_cluster(cluster_config,
                                     imported=True)
 
+# ==============================
+# EKS launch template test cases
+
+
+@ekscredential
+def test_eks_v2_hosted_cluster_create_basic_launch_template():
+    """
+    Create a hosted EKS v2 cluster with all default values from the UI
+    This does not use a preexisting launch template.
+    """
+    cluster_name = random_test_name("test-auto-eks")
```

Comment:
This is a basic EKS cluster create using launch templates. We currently do not have a check in place to differentiate  cluster creation in v2.5.5 vs v2.5.6.

Reply:
If so, can we add this in the next PR ?